doc/modules/partial_dependence.rst

```diff
@@ -112,17 +175,22 @@ values are defined by :math:`x_S` for the features in :math:`X_S`, and by
 :math:`x_C` for the features in :math:`X_C`. Note that :math:`x_S` and
 :math:`x_C` may be tuples.
 
-Computing this integral for various values of :math:`x_S` produces a plot as
-above.
+Computing this integral for various values of :math:`x_S` produces a PDP plot
+as above.
+
+In ICE, for each sample in :math:`\{(X_S^{(i)}, X_C^{(i)})\}_{i=1}^N` the
+line :math:`pd_{X_S}^{(i)}` is calculated against :math:`X_S^{(i)}`, while
+:math:`X_C^{(i)}` remains fixed.
```

Comment:
The use of the term "sample" is a bit confusing here. I would remove this part, and define ICEs below as suggested. Maybe we can just add

"The ICE values are defined below"